content/sensu-go/6.2/operations/deploy-sensu/datastore.md

```diff
@@ -496,3 +499,4 @@ enable_round_robin: true
 [18]: #datastore-specification
 [19]: ../install-sensu/#ports
 [20]: https://www.postgresql.org/docs/current/config-setting.html
+[21]: https://etcd.io/docs/v3.3.13/op-guide/clustering/
```

Comment:
This link is out of date, we should update it to the 3.5 documentation. Also, I believe "clustering" is probably the wrong documentation section, there is likely another section on authorization.

Reply:
Doc link: https://etcd.io/docs/v3.5/op-guide/authentication/

Reply:
Thanks @echlebek -- this PR is quite a bit behind, so those links are already fixed in the published content. I'll need to be careful of that kind of thing though when I finally do merge this, so thank you.

Is this the only thing you found? Or still testing?

Reply:
I'm still testing... it's taking me a while to get the environment set up properly. Sorry this is taking so long.

Reply:
Not at all, I really appreciate your help!